### PR TITLE
migrate Lighthouse content from d.g.c to d.c.c

### PR DESCRIPTION
**Pull request by brendankenny**
Part of #2008 (remaining task will be adding the redirects on the WebFundamentals side)

- overview: With Lighthouse 10.0 coming soon, we will have more docs changes coming, so I only spent a little time updating this content: new screenshots, new icons, and fixed anything blatantly inaccurate (e.g. menu items that don't exist anymore, etc). Otherwise, content is largely the same.
- v3-migration-guide: this doc is four years old now, so not going to link it from the main ToC, however it is linked from other content (e.g. https://developer.chrome.com/blog/lighthouse3/), so I migrated it to not contribute to link rot

The only other [Lighthouse content on Web Fundamentals](https://github.com/google/WebFundamentals/tree/main/src/content/en/tools/lighthouse):
- this [`run` page](https://github.com/google/WebFundamentals/blob/c676c1eb5ffa169479ae133bb70cda990a4bfda7/src/content/en/tools/lighthouse/run.md), which I don't totally understand (looks like it once had an iframed precursor to web.dev/measure?), but it does show up in search results, so I vote add a redirect to https://pagespeed.web.dev/ and call it a day
- [`variability`](https://github.com/google/WebFundamentals/blob/c676c1eb5ffa169479ae133bb70cda990a4bfda7/src/content/en/tools/lighthouse/variability.md) - I'm not really sure why this exists. There's a [copy of it in the lighthouse repo](https://github.com/GoogleChrome/lighthouse/blob/94b4f38d53694d6fdec38a12481f5dbd6a856f08/docs/variability.md), and that one has gotten updated more recently. The lighthouse repo doc is also the one we regularly link people to in Lighthouse issues/twitter/etc, so I'm good with a redirect to it. In the future it might make more sense for that content to live on d.c.c, but we can migrate it once we have a plan and update the redirect at that point.

The last bit of content not included is translations of the overview. Most of it dates from 2017 (https://github.com/google/WebFundamentals/pull/4430), so is quite different from the currently published content. It didn't seem worth bringing over given that we're likely going to be reworking the overview in the next month or two for Lighthouse 10.0 anyways, but let me know if others have a different opinion on that.

### CODE REVIEW

File: site/en/docs/lighthouse/overview/index.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Overview
+description: |
+  Learn how to set up Lighthouse to audit your web apps.
+date: 2016-09-27
+updated: 2022-05-24
+---
+
+Lighthouse is an [open-source](https://github.com/GoogleChrome/lighthouse), automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, SEO and more.
```
Comment (malchata) [inline site/en/docs/lighthouse/overview/index.md:10]:
```suggestion
[Lighthouse](https://github.com/GoogleChrome/lighthouse) is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, SEO and more.
```

File: site/en/docs/lighthouse/overview/index.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/QXJ5lmcjHEFLTHg5B4o8.svg", class="float-right", alt="Lighthouse logo", width="150", height="150" %}
+
+You can run Lighthouse in Chrome DevTools, from the command line, or as a Node module. You give Lighthouse a URL to audit, it runs a series of audits against the page, and then it generates a report on how well the page did. From there, use the failing audits as indicators on how to improve the page. Each audit has a reference doc explaining why the audit is important, as well as how to fix it.
+
+You can also use [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/getting-started.md) to prevent regressions on your sites.
+
+Check out the video below from Google I/O to learn more about how to use and contribute to Lighthouse.
+
+{% YouTube id="mLjxXPHuIJo" %}
```
Comment (malchata) [inline site/en/docs/lighthouse/overview/index.md:20]:
```suggestion
<figure>
  {% YouTube id="mLjxXPHuIJo" %}
</figure>
```

**Review by malchata — APPROVED** [review]
LGTM!

**Comment by malchata** [comment]
@brendankenny, I will leave it to you to decide when to merge, since there's a few moving parts here.

File: site/en/docs/lighthouse/overview/index.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+* [As a Node module](#programmatic). Integrate Lighthouse into your continuous integration systems.
+* [From a web UI](#psi). Run Lighthouse and link to reports without installing a thing.
+
+{% Aside %}
+The CLI and Node workflows require you to have an instance of Google Chrome installed on your machine.
+{% endAside %}
+
+### Run Lighthouse in Chrome DevTools {: #devtools }
+
+Lighthouse powers the **Audits** panel of Chrome DevTools. To run a report:
```
[the remark this answers is not in the document]
Comment (brendankenny) [reply site/en/docs/lighthouse/overview/index.md:37]: > This is no longer true, as that panel has been renamed to Lighthouse.

Good catch! Rephrasing this sentence was going to be awkward so I meant to come back to it, but forgot :)

**Comment by brendankenny** [comment]
I think we should be good to land since lighthouse is still not listed on the main docs page. The real tricky bit will be the WebFundamentals redirects.

**Pull request merged**